scenarioo-client/app/manage/generalSettings/generalSettings.controller.js

```diff
@@ -17,13 +17,14 @@
 
 angular.module('scenarioo.controllers').controller('GeneralSettingsController', GeneralSettingsController);
 
-function GeneralSettingsController($scope, BranchesResource, ConfigService) {
+function GeneralSettingsController($scope, BranchesResource, ConfigService, SearchEngineStatusService) {
 
     var vm = this;
     vm.branches = [];
     vm.configuration = {};
     vm.configuredBranch = {};
     vm.successfullyUpdatedConfiguration = false;
+    vm.isEngineRunning = false;
```

Comment:
Engine - what kind of engine ?? please rename to something like "isSearchEngineRunning"